src/Transition/CreateNewIDFUsingRulesV9_3_0.f90

```diff
@@ -1845,4 +1875,139 @@ SUBROUTINE SortUnique(StrArray, Size, Order)
     END DO
   END DO
 
-END SUBROUTINE SortUnique
\ No newline at end of file
+END SUBROUTINE SortUnique
+
+SUBROUTINE UpdateEMSFunctionName(InOutArg, OldName, NewName, NoDiffArg)
+  USE InputProcessor, ONLY: SameString
+  CHARACTER(len=*), INTENT(INOUT) :: InOutArg
+  CHARACTER(len=100) ::  NewInOutArg
+  CHARACTER(len=*), INTENT(IN) :: OldName
+  CHARACTER(len=*), INTENT(IN) :: NewName
+  LOGICAL, INTENT(INOUT) :: NoDiffArg
+
+  PARAMETER(N=20)
+  CHARACTER*20 ARRAY(N)
```

Comment:
So this will allow up to 20 tokens to come back?

Reply:
It's just a guess at a conservative number.

Reply:
Yeah, I mean given the line length limit that was put on IDFs, this seems reasonable.